src/screens/SearchBeers/SearchBeers.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import { fetchAllBeers } from '../../services/beers';
+import Button from '../../shared/components/Button/Button';
+import BeersList from '../../shared/components/BeersList/BeersList';
+import './styles.scss';
+import Heading from '../../shared/components/Heading/Heading';
+
+class SearchBeersComponent extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beers: null,
+      smallBeer: null
+    }
+  }  
+
+  getAllBeers = async () => {
+    const allBeers = await fetchAllBeers()
+    this.setState({
+      beers: allBeers,
+    })
+  }
+
+  weakBeers = () => {
+    const smallvolumeAlcohol = this.state.beers.filter(beer=> beer.abv <= 4.5);
+    this.setState({
+      smallBeer: smallvolumeAlcohol,
+    })
+  }
+  
+  render(){
+    const beers = this.state.beers;
+    return (
+      <div className="search-page">
+        <Heading 
+          type="h3"
+          className="title-search"
+          text='Find Beers'
+        />
+        <Heading 
+          text="Filter"
+          type="h4" />
+        <div className="filter-select">
+          <select className="option">
+            <option>--Alcohol volume--</option>
+            <option onClick={this.weakBeers}>less 4,5 %</option>
```

Comment:
So here it is weak beer, there it was small...

Reply:
Done